### PR TITLE
Add model, migration, and factory for package

### PR DESCRIPTION
**Pull request by JeffreyDavidson**

### CODE REVIEW

File: src/Models/Note.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Tipoff\Notes\Models;
```
Comment (drewroberts) [inline src/Models/Note.php:3]: There is a typo here, Models should be before Notes in the namespace.

File: database/factories/NoteFactory.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Database\Factories;
```
Comment (drewroberts) [inline database/factories/NoteFactory.php]: Tipoff/Notes/Database/Factories should be the namespace on this file

File: src/Models/Note.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Note extends Model
```
Comment (drewroberts) [inline src/Models/Note.php]: We should extend the BaseModel from the support package

File: src/Models/Note.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+{
+    use HasFactory;
```
Comment (drewroberts) [inline src/Models/Note.php]: Instead of HasFactory, we need to use the HasPackageFactory trait

**Review by drewroberts — COMMENTED** [review]
We should also use .stub extensions in packages on the migration files

File: src/Models/Note.php (hunk 1 above)
Comment (JeffreyDavidson) [reply src/Models/Note.php:3]: @drewroberts Can you explain this to me? The composer.json says the package namespace is Tipoff\Notes so shouldn't the namespace be Tipoff\Notes and then the name of the directory in this case Models?

File: src/Models/Note.php (hunk 4 above)
Comment (JeffreyDavidson) [reply src/Models/Note.php]: @drewroberts I can't find a package where the models use that trait. All of them extend the HasFactory trait as well. Is this something will need to be changed in all of the existing packages as well.

**Comment by JeffreyDavidson** [comment]
> We should also use .stub extensions in packages on the migration files

@drewroberts Can we schedule a quick call for you to show me what you mean here?

File: src/Models/Note.php (hunk 1 above)
Comment (drewroberts) [reply src/Models/Note.php:3]: Haha, the typo was in my head. You are correct. My bad!

File: src/Models/Note.php (hunk 4 above)
Comment (drewroberts) [reply src/Models/Note.php]: Thank you. When you get a chance, run a PR on those other packages to update them as well.

**Comment by drewroberts** [comment]
> > We should also use .stub extensions in packages on the migration files
> 
> @drewroberts Can we schedule a quick call for you to show me what you mean here?

Yep, I'll shoot you a message on Slack

**Review by drewroberts — APPROVED** [review]
Thank you, Jeffrey! 🤴

**Pull request merged**